src/core/RuntimeErrorCollector.cpp

```diff
@@ -98,7 +98,7 @@ int RuntimeErrorCollector::count() const {
   int totalMessages;
   const int numMessages = m_errors.size();
 
-  all_reduce(m_comm, numMessages, totalMessages, std::plus<int>());
+  all_reduce(m_comm, numMessages, totalMessages, std::plus<>());
```

Comment:
Can you check that boost.mpi can still detect that this is a MPI build-in operation with this?

Reply:
There are multiples like this.

Reply:
How can I check that?

Reply:
Look at the implementation

Reply:
I can look it up

Reply:
Looking at https://github.com/boostorg/mpi/blob/develop/include/boost/mpi/operations.hpp#L218, `std::plus<void>` is not yet recognized...

Reply:
FYI: https://github.com/boostorg/mpi/pull/83, but that of course does not help us now.

Reply:
does this has a significant impact on performance?

Reply:
Yes